projects/1.21/assets/ender-io/enderio/lang/zh_cn.json

```
@@ -0,0 +1,877 @@
+    "block.enderio.fused_quartz_yellow": "黄色石英玻璃",
+    "block.enderio.hootch": "烈酒",
+    "block.enderio.industrial_insulation_block": "工业隔离方块",
+    "block.enderio.liquid_darkness": "液态黑暗",
+    "block.enderio.liquid_sunshine": "液态阳光",
+    "block.enderio.nutrient_distillation": "营养精华液",
+    "block.enderio.painted_crafting_table": "喷涂过的工作台",
+    "block.enderio.painted_fence": "喷涂过的栅栏",
+    "block.enderio.painted_fence_gate": "喷涂过的栅栏门",
+    "block.enderio.painted_glowstone": "喷涂过的荧石",
+    "block.enderio.painted_redstone_block": "喷涂过的红石块",
+    "block.enderio.painted_sand": "喷涂过的沙子",
+    "block.enderio.painted_slab": "喷涂过的台阶",
+    "block.enderio.painted_stairs": "喷涂过的楼梯",
+    "block.enderio.painted_trapdoor": "喷涂过的活板门",
+    "block.enderio.painted_wall": "喷涂过的墙",
+    "block.enderio.painted_wooden_pressure_plate": "喷涂过的木压力板",
+    "block.enderio.pulsating_alloy_block": "脉冲合金块",
+    "block.enderio.redstone_alloy_block": "红石合金块",
+    "block.enderio.reinforced_obsidian_block": "强化黑曜石",
+    "block.enderio.resetting_lever_five": "复位拉杆（5秒）",
+    "block.enderio.resetting_lever_five_inv": "反相复位拉杆（5秒）",
+    "block.enderio.resetting_lever_sixty": "复位拉杆（1分钟）",
+    "block.enderio.resetting_lever_sixty_inv": "反相复位拉杆（1分钟）",
+    "block.enderio.resetting_lever_ten": "复位拉杆（10秒）",
+    "block.enderio.resetting_lever_ten_inv": "反相复位拉杆（10秒）",
+    "block.enderio.resetting_lever_thirty": "复位拉杆（30秒）",
+    "block.enderio.resetting_lever_thirty_inv": "反相复位拉杆（30分钟）",
+    "block.enderio.resetting_lever_three_hundred": "复位拉杆（5分钟）",
+    "block.enderio.resetting_lever_three_hundred_inv": "反相复位拉杆（5分钟）",
+    "block.enderio.rocket_fuel": "火箭燃料",
+    "block.enderio.silent_acacia_pressure_plate": "静音金合欢木压力板",
+    "block.enderio.silent_birch_pressure_plate": "静音白桦木压力板",
+    "block.enderio.silent_crimson_pressure_plate": "静音绯红木压力板",
+    "block.enderio.silent_dark_oak_pressure_plate": "静音深色橡木压力板",
+    "block.enderio.silent_dark_steel_pressure_plate": "静音玄钢压力板",
+    "block.enderio.silent_heavy_weighted_pressure_plate": "静音重质测重压力板",
+    "block.enderio.silent_jungle_pressure_plate": "静音丛林木压力板",
+    "block.enderio.silent_light_weighted_pressure_plate": "静音轻质测重压力板",
+    "block.enderio.silent_oak_pressure_plate": "静音橡木压力板",
+    "block.enderio.silent_polished_blackstone_pressure_plate": "静音磨制黑石压力板",
+    "block.enderio.silent_soularium_pressure_plate": "静音魂金压力板",
+    "block.enderio.silent_spruce_pressure_plate": "静音云杉木压力板",
+    "block.enderio.silent_stone_pressure_plate": "静音石头压力板",
+    "block.enderio.silent_warped_pressure_plate": "静音诡异木压力板",
+    "block.enderio.soul_chain": "灵魂链条",
+    "block.enderio.soularium_block": "魂金块",
+    "block.enderio.soularium_pressure_plate": "魂金压力板",
+    "block.enderio.vapor_of_levity": "轻盈之气",
+    "block.enderio.vibrant_alloy_block": "振动合金块",
+    "block.enderio.void_chassis": "虚空框架",
+    "block.enderio.xp_juice": "经验汁",
+    "enderio_base.configuration.blinkRange": "闪烁范围",
```

```suggestion
    "enderio_base.configuration.blinkRange": "闪现范围",
```
https://wiki.leagueoflegends.com/en-us/Blink
学习到